lib/cdo/pegasus/graphics.rb

```diff
@@ -2,20 +2,100 @@
 
 MAX_DIMENSION = 2880
 
-def load_manipulated_image(path, mode, width, height = nil)
+def load_manipulated_image(path, mode, width, height, scale = nil)
```

Comment:
I checked that no existing callers were making use of the default `height` parameter, so i don't think we still need it